definitions/windows.yaml

```diff
@@ -443,6 +594,17 @@ labels: [Software]
 supported_os: [Windows]
 urls: ['http://www.forensicswiki.org/wiki/Windows_Desktop_Search']
 ---
+name: WindowsSecurityProviders
+doc: Security Providers DLLs
+sources:
+- type: REGISTRY_VALUE
+  attributes:
+    key_value_pairs: [{key: '%%current_control_set%%\Control\SecurityProviders', value: 'SecurityProviders'}]
```

Comment:
I copied this from existing examples. Once @destijl provides an answer, I'll update if required.